apps/orkestratoren/orkestratoren-core/src/main/java/no/nav/registre/orkestratoren/config/SwaggerConfig.java

```diff
@@ -15,8 +15,7 @@
 import springfox.documentation.spring.web.plugins.Docket;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
-import java.util.Arrays;
-import java.util.HashSet;
+import java.util.*;
```

Comment:
Vi unngår å bruke *-imports, fordi det er lettere å lese kode der alle imports er gitt eksplisitt. Det er en innstilling i Intellij: Settings -> Editor -> Code Style -> Java ->'Class count to use import with' og 'Names count to use static import with'. Sett disse to til et høyt tall, f.eks 99.

Reply:
Skjønner, lurte litt på den ja. Takk for tipset!